app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -7,13 +7,12 @@ def build_for_importer
       build_metadata
       return false unless collections_created?
       begin
-        @item = factory.run
+        factory.run
+        return true
       rescue StandardError => e
         status_info(e)
-      else
-        status_info
```

Comment:
you are no longer clearing the errors once an entry succeeds

Reply:
reinstated